### PR TITLE
SEP-9: add `clabe_number`

### LINKED ISSUE #1201: SEP-9: add `clabe_number` to list of standardized fields
### What problem does your feature solve?

Mexico's banking system makes heavy use of [CLABE numbers](https://bank.codes/guides/clabe-numbers-guide), but our standard does not support this value format.

### What would you like to see?

Add a `clabe_number` attribute to the standardized list.

### What alternatives are there?

CLABE numbers are composed of values, some of which are already standardized in SEP-9. These values are:

- `bank_branch`
- `bank_account_number`

However, we are missing attributes to describe the bank itself, as well as a "validation digit".

Given the added complexity of generating the CLABE number from a set of values passed via SEP-9 attributes, I would recommend adding a single `clabe_number` attribute instead, even if there is technically some overlap with the attributes we have already defined.

cc @leighmcculloch, what do you think?

### PR DESCRIPTION
**Pull request by JakeUrban**
resolves #1201

### CODE REVIEW

File: ecosystem/sep-0009.md [hunk 1]
```diff
@@ -109,3 +110,6 @@ Name | Type | Description
 
 Address formatting varies widely from country to country and even within each country. See [here](https://stackoverflow.com/questions/11160192/how-to-parse-freeform-street-postal-address-out-of-text-and-into-components) for details. Rather than attempting to create a field for each possible part of an address in every country, this protocol takes a middle of the road approach. Address fields that are fairly universal can be encoded with the `country_code`, `state_or_province`, `city`, and `postal_code` fields. Full addresses, however, should be encoded as a single multi-line string in the `address` field. This allows any address in the world to be represented with a limited number of fields. If address parsing is necessary, parsing will be easier since the country, city, and postal code are already separate fields.
 
+## Changelog
+
+* `v1.6.0`: add `clabe_number` to Natural Person KYC fields ([#1202](https://github.com/stellar/stellar-protocol/pull/1202))
```
Comment (leighmcculloch) [inline ecosystem/sep-0009.md:115]:
```suggestion
* `v1.6.0`: Add `clabe_number` to Natural Person KYC fields ([#1202](https://github.com/stellar/stellar-protocol/pull/1202)).
```

**Review by leighmcculloch — COMMENTED** [review]
Looks good, one typo:

**Review by leighmcculloch — APPROVED** [review]

**Comment by nirojapanda** [comment]
Should we be adding this for every country or is there a way to standardize it?

**Comment by JakeUrban** [comment]
@nirojapanda `clabe_number` is composed of related values, some of which are already standardized in SEP-9. CLABE numbers are specific to brazil though.

**Pull request merged**